src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+
+    String From;
+    String To;
+    public Event(String description, String From, String To) {
```

Comment:
Perhaps, the naming of the fields can follow the norm of the 'description' field with small letters instead of capitalised ones. I think as long as you are consistent, it should be fine